README.md

```
@@ -1,240 +1,235 @@
 
-### "I only want today's photos"
-```bash
-./stackcopy.py --copy /photos/camera-import /photos/today-stacked --today
-```
+When using `--lightroom` or `--lightroomimport`, stack input frames are moved to:
 
-### "Add a custom prefix to the renamed files"
-```bash
-./stackcopy.py --stackcopy /photos/mushrooms --prefix "Jackson State Forest"
-# Creates files like: "IMG_1234 Jackson State Forest stacked.jpg"
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)